nautobot/docs/models/extras/relationship.md

```diff
@@ -9,11 +9,40 @@ Sometimes it is desirable to create a new kind of relationship between one (or m
 
 ## Relationship Filters
 
-Filters can be defined to restrict the type or selection of objects for either side of the connection. From the FEX example above, you can restrict the FEX side of the connection to only of devices with the FEX role, and restrict the controller side to only devices with Controller roles.
+Filters can be defined to restrict the type or selection of objects for either side of the connection. 
+
+### Relationship Filter Example
+
+As an example, let's create a relationship between Circuits and Devices. 
+In our situation we only would terminate Circuits on Devices with the Role of `edge`.
+
+To prevent the Circuit Relationship from showing up on any other Device, use a JSON filter to 
+limit the Relationship to only Devices with Role of `edge`:
+
```

Comment:
Might be helpful somewhere in here to explain how to determine what filters are available for a given model:

- by reading the code (here, `nautobot/dcim/filters.py` -> `DeviceFilterSet` class)
- by checking the filter options available in the REST API (e.g., `http://.../api/docs` and in this case checking the `dcim_devices_list` API endpoint)

Reply:
love this example because I learned something